### PR TITLE
Display NIM currency symbol before balance

### PR DESCRIPTION
**Pull request by sisou**
## Pull request checklist

- [x] All tests pass. Demo project builds and runs.
- [x] I have resolved any merge conflicts. Confirmation: ____

## What's in this pull request?
* Display NIM currency symbol in browser client

Please see https://soerenschwert.de/nimiq/clients/browser/ for a demonstration of this PR.

### CODE REVIEW

**Comment by sisou** [comment]
Please see https://soerenschwert.de/nimiq/clients/browser/ for a demonstration of this PR.

File: clients/browser/index.html [hunk 1]
```diff
@@ -404,7 +404,7 @@
 
         _balanceChanged(balance) {
             if (!balance) balance = Nimiq.Balance.INITIAL;
-            this._wltBalance.innerText = Nimiq.Policy.satoshisToCoins(balance.value).toFixed(Math.log10(Nimiq.Policy.SATOSHIS_PER_COIN));
+            this._wltBalance.innerHTML = '<span class="icon-nimiq"></span> ' + Nimiq.Policy.satoshisToCoins(balance.value).toFixed(Math.log10(Nimiq.Policy.SATOSHIS_PER_COIN));
```
Comment (paberr) [inline clients/browser/index.html]: I would prefer keeping `innerText` here and not putting the icon there on every update.
Instead, you could hardcode the icon into the HTML right before the balance, e.g., as follows:
```
<span class="icon-nimiq"></span> <span id="wltBalance">0.00000000</span>
```

Comment (sisou) [inline clients/browser/index.html]: My problem with that was that the currency symbol then even displayed during initialization when no balance (not even 0.0000) is displayed. It looked odd. But that's my only reason :D

Comment (mar-v-in) [inline clients/browser/index.html]: Shouldn't it work to just add the `icon-nimiq` class to the html element once you want to display the balance?

Comment (paberr) [inline clients/browser/index.html]: At least for me, this has some strange effects on the font of the following balance. But I didn't really look into the CSS, I just did a quick test.

Comment (mar-v-in) [inline clients/browser/index.html]: That's because the `icon-*` rule should better be a `icon-*:before` rule so it doesn't affect unrelated stuff...

Comment (sisou) [inline clients/browser/index.html]: I "repaired" this problem with my latest commits. Please review again.

**Comment by sisou** [comment]
Any thoughts on this?

**Comment by paberr** [comment]
Thank you @sisou ! I'll merge it.
